cli/pkg/validate/required_keys.go

```diff
@@ -319,7 +319,7 @@ func (rk *RequiredKeysValidator) validateNumberConfig(config map[string]any, ref
 	numericFields := []string{"minimum", "maximum", "exclusiveMinimum", "exclusiveMaximum", "multipleOf"}
 	for _, field := range numericFields {
 		if val, ok := config[field]; ok {
-			if !isNumber(val) {
+			if !isInteger(val) {
```

Comment:
numericfields like maximum can have number values eg: 3.5, 5.7, so on..
isInteger should be validated if data type is integer